### ISSUE TITLE
Suppress tab bar width issue

### ISSUE THREAD

**Issue by crathecrub** [anakinsk/SafariStand#5]
Suppress tab bar width is not working

Another issues, is if you check the 'Suppress Tab Bar Width' option, when you click a tab, a new one pops out. The only fix is disable suppress tab bar width.

I'm hoping Apple has just renamed a class or something?

**Comment by anakinsk** [comment]
Hey how is that suppose to work exactly, like setting a width of the tab to some particular value and preventing safari from changing it in either direction?

**Comment by crathecrub** [comment]
Yep :) What it will do is set a min/maximum with so you can view many tabs without scrolling through them.
